### PR TITLE
SC-225235 lineage cell tooltip for mpox

### PR DESCRIPTION
**Pull request by ehoops-cz**
### Summary:
- **What:** `Add lineage info tooltip for mpox. Add config to swap between components.  Minor fix for covid lineage tooltip to properly hide unused fields`
- **Ticket:** [sc225235](https://app.shortcut.com/genepi/story/225235)
- **Env:** `none`

### Demos:
Screenshot 2022-12-14 at 11 52 27 AM
Screenshot 2022-12-14 at 11 52 16 AM

### Checklist:
- [x] I merged latest `<base branch>`
- [x] I manually verified the change
- [x] I added labels to my PR
- [ ] I tested in multiple browsers
- [ ] I added relevant unit tests
- [ ] I have notified others of changes they need to make locally (migrations, jobs, package updates, etc)

### CODE REVIEW

File: src/frontend/src/views/Data/components/SamplesView/components/SamplesTable/components/LineageTooltip/components/CovidLineageTooltip/index.tsx [hunk 1]
```diff
@@ -0,0 +1,70 @@
+import { Tooltip } from "czifui";
```
Comment (ehoops-cz) [inline src/frontend/src/views/Data/components/SamplesView/components/SamplesTable/components/LineageTooltip/components/CovidLineageTooltip/index.tsx:1]: This file is the original tooltip component

File: src/frontend/src/views/Data/components/SamplesView/components/SamplesTable/components/LineageTooltip/components/CovidLineageTooltip/index.tsx [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+      {DISPLAY_ORDER.map((key) => {
+        const value = lineage[key];
+        // skip certain keys for now that are extra and not included in current design
+        if (
+          ![
+            "reference_dataset_name",
+            "reference_sequence_accession",
+            "reference_dataset_tag",
+            "lineage_type",
+          ].includes(key)
```
Comment (ehoops-cz) [inline src/frontend/src/views/Data/components/SamplesView/components/SamplesTable/components/LineageTooltip/components/CovidLineageTooltip/index.tsx:48]: This is the only change.  These fields were not properly hiding before.

File: src/frontend/src/views/Data/components/SamplesView/components/SamplesTable/components/LineageTooltip/components/GeneralViralLineageTooltip/index.tsx [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+  "reference_dataset_name",
+  "lineage_software_version",
+  "lineage_type",
+  "lineage_probability",
+  "last_updated",
+  "scorpio_call",
+  "scorpio_support",
+  "reference_sequence_accession",
+  "reference_dataset_tag",
+];
```
Comment (ehoops-cz) [inline src/frontend/src/views/Data/components/SamplesView/components/SamplesTable/components/LineageTooltip/components/GeneralViralLineageTooltip/index.tsx:31]: I'm using the same pattern from the existing tooltip of listing all of the fields and then hiding the ones we don't need.  I'm not sure if this is confusing and/or why it was done this way in the first place.  Thought?

File: src/frontend/src/views/Data/components/SamplesView/components/SamplesTable/components/LineageTooltip/components/GeneralViralLineageTooltip/index.tsx [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+const KEY_TO_LABELS = {
+  lineage: "Lineage",
+  qc_status: "QC Status",
+  scorpio_call: "Scorpio Call",
+  scorpio_support: "Scorpio Support",
+  lineage_software_version: "Version",
+  lineage_type: "Lineage Type",
+  lineage_probability: "Lineage Probability",
+  last_updated: "Last Updated",
+  reference_dataset_name: "Nextclade Dataset",
```
Comment (ehoops-cz) [inline src/frontend/src/views/Data/components/SamplesView/components/SamplesTable/components/LineageTooltip/components/GeneralViralLineageTooltip/index.tsx:14]: - [x]  Can someone please confirm that this is the correct field to use for Nextclade dataset?

**Review by jgadling — APPROVED** [review]
I'm rubberstamp'ing this. If you think it's fairly safe to merge, go ahead and get it on staging.

File: src/frontend/src/views/Data/components/SamplesView/components/SamplesTable/components/LineageTooltip/components/GeneralViralLineageTooltip/index.tsx (hunk 4 above)
Comment (lvreynoso) [reply src/frontend/src/views/Data/components/SamplesView/components/SamplesTable/components/LineageTooltip/components/GeneralViralLineageTooltip/index.tsx:14]: Looks right to me - see https://github.com/chanzuckerberg/czgenepi/blob/c1370202b18134631696ac1ba52cc6eb17bcbad4/src/backend/aspen/database/models/lineages.py#L113...L121

Comment (ehoops-cz) [reply src/frontend/src/views/Data/components/SamplesView/components/SamplesTable/components/LineageTooltip/components/GeneralViralLineageTooltip/index.tsx:14]: thanks!

**Pull request merged**